### PR TITLE
feat: adding GitHub info

### PR DESCRIPTION
**Pull request by benarmstrongbf**

### CODE REVIEW

File: pages/installation.md [hunk 1]
```diff
@@ -16,17 +16,304 @@ permalink: /installation/
 
 _Ensure you have read the [Prerequisites](/installation#Prerequisites) section._
 
-Installing via Github requires you to download all the microservice repositories individually.
+### Prerequisites for Github
 
-_This documentation site is a work in progress. Check back for updates._
+- yarn 1.22.x (npm install -g yarn)
+- Go [1.15.x](https://golang.org/doc/install)
+- Ruby [2.6.6](https://www.ruby-lang.org/en/downloads/) (with [bundler](https://bundler.io/) installed)
```
Comment (jackregnart) [inline pages/installation.md]: Ruby 2.7.1 is now required.

File: pages/installation.md [hunk 2]
```diff
@@ -16,17 +16,304 @@ permalink: /installation/
...
+sudo rabbitmqctl add_user oop somepassword
+sudo rabbitmqctl list_permissions --vhost /
+sudo rabbitmqctl set_permissions -p / oop ".*" ".*" ".*"
+```
+
+Ensure this is reflected in `OOP_AMQP_ADDRESS` connection string environment variable.
+
+##### Import the queue config
+
+There is a pre-configured queue config in the [oop-queue-config](https://github.com/open-interop/oop-queue-config) repository. Download `rabbit-config.json` from the repo and run `sudo rabbitmqadmin import rabbit-config.json` to import it.
```
Comment (jackregnart) [inline pages/installation.md]: @danyoung93 is this still a requirement?

File: pages/installation.md [hunk 3]
```diff
@@ -16,17 +16,304 @@ permalink: /installation/
...
+POSTGRES_DB=oop_core_development
+
+SECRET_KEY_BASE={SECRETKEY}
+```
+
+### oop-core
+
+Clone the application `git clone http://github.com/open-interop/oop-core.git`
+In the app directory install all dependencies using  `bundle install`
+Create a symlink between
```
Comment (jackregnart) [inline pages/installation.md]: What's going on here?

File: pages/installation.md [hunk 4]
```diff
@@ -16,17 +16,304 @@ permalink: /installation/
...
+production:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+```
+
+
+Create the database - `bundle exec rails db:create`
+Run the database migrations - `bundle exec rails db:migrate`
+
+
+To allow login from the interface run the following commands from within the `oop-core` directory to create a dummy user:
```
Comment (jackregnart) [inline pages/installation.md]: Would be good to refer to the script that I made to do this.

File: pages/installation.md [hunk 5]
```diff
@@ -16,17 +16,304 @@ permalink: /installation/
...
+
+The following services can all be installed and set up in exactly the same way, cloning the repo and using yarn to install any dependencies. 
+
+#### oop-gateway
+
+`git clone http://github.com/open-interop/oop-gateway.git`
+
+`cd oop-gateway`
+
+`ln ../.env`
```
Comment (jackregnart) [inline pages/installation.md]: i'd prefer explicit `ln -nfs /explicit/path /explicit/path` just so there's no confusion.

**Review by jackregnart — CHANGES_REQUESTED** [review]

File: pages/installation.md (hunk 4 above)
Comment (benarmstrongbf) [reply pages/installation.md]: Whereabouts can I find this?

Comment (jackregnart) [reply pages/installation.md]: If you run `bin/rails open_interop:setup_initial_account` it runs you through some options.

File: pages/installation.md [hunk 6]
```diff
@@ -8,25 +8,302 @@ permalink: /installation/
...
+
+When all microservices are installed and core is up, run `yarn start`
+
+#### oop-tempr
+
+`git clone http://github.com/open-interop/oop-tempr.git`
+
+`cd oop-tempr`
+
+`ln -nfs /path/to/.env /path/to/oop-tempr`
```
Comment (jackregnart) [inline pages/installation.md]: Maybe something like this would make more sense?

`ln -nfs /path/to/.global-env /path/to/oop-tempr/.env`

**Review by jackregnart — APPROVED** [review]

**Pull request merged**